js/src/forum/components/Post.js

```diff
@@ -125,7 +126,7 @@ export default class Post extends Component {
       classes.push('Post--by-actor');
     }
 
-    if (user && app.current.discussion && app.current.discussion.attribute('startUserId') == user.id()) {
+    if (user && user == discussion.user()) {
```

Comment:
If I am not mistaken, the previous code worked even when the discussion's user object has not been loaded. Can we guarantee that?

Reply:
Same for navigating from the post to its `discussion()` - do we reverse-fill that relationship?

Reply:
> If I am not mistaken, the previous code worked even when the discussion's user object has not been loaded. Can we guarantee that?

Not sure, we could change this back though? It worked for me when I tested locally.



> Same for navigating from the post to its `discussion()` - do we reverse-fill that relationship?

Where would this be?

Reply:
Does it lazy-load the data? :thinking:

Reply:
Talked it through. Resolved.